pkg/mimirtool/commands/analyse_grafana_test.go

```diff
@@ -43,6 +47,7 @@ func TestParseMetricsInBoard(t *testing.T) {
 
 	analyze.ParseMetricsInBoard(output, board)
 	assert.Equal(t, dashboardMetrics, output.Dashboards[0].Metrics)
+	assert.Equal(t, expectedParseErrors, output.Dashboards[0].ParseErrors)
```

Comment:
This would have caught the issue in my previous PR, but since we were not asserting that there are no unexpected errors, it slipped by!